docs/pages/faq/clear-cache-macos-linux.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Clearing bundler caches on macOS and Linux
+---
+
+> Need to clear development caches on Windows? [Find the relevant commands here.](clear-cache-windows.md)
+
+There are a number of different caches associated with your project that can prevent your project from running as intended. Clearing a cache sometimes can help you work around issues related to stale or corrupt data and is often useful when troubleshooting and debugging.
+
+To clear the various caches, run:
```

Comment:
we should just make a cli tool that does all of these things

Reply:
lol, `expo forget`